lib/version_utils.pm

```diff
@@ -376,7 +376,7 @@ sub is_staging {
 Returns true if storage_ng is used
 =cut
 sub is_storage_ng {
-    return get_var('STORAGE_NG') || is_sle('15+') || is_tumbleweed;
+    return get_var('STORAGE_NG') || is_sle('15+');
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You already remove `is_tumbleweed` but you can also try to negate the logic check.

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
yes, I got your point, it is a good remainder, but the best solution is to not use this sort of conditions at all. This function was created wrongly without the default you mentioned of 1 and the exceptions and collaborators just pile up more conditions without having scope of all products.